tests/app/dao/test_users_dao.py

```diff
@@ -36,62 +38,55 @@ def test_create_user(notify_api, notify_db, notify_db_session):
     assert not user.platform_admin
 
 
-def test_get_all_users(notify_api, notify_db, notify_db_session, sample_user):
-    assert User.query.count() == 1
-    assert len(get_user_by_id()) == 1
-    email = "another.notify@digital.cabinet-office.gov.uk"
-    another_user = create_sample_user(notify_db,
-                                      notify_db_session,
-                                      email=email)
+@pytest.mark.usefixtures('notify_db_session')
```

Comment:
i'm not entire sold on this `usefixtures` thing - my aim with using it is to signify that "this is a fixture with side effects" as opposed to "this is a fixture that gives you something useful" (and also to supress linter warnings about unused functions)

thoughts?